documentation/book/con-product-downloads.adoc

```diff
@@ -3,14 +3,23 @@
 // getting-started.adoc
 
 [id='downloads-{context}']
-= {ProductName} downloads
+= Installing {ProductName} and deploying components
 
 ifdef::Downloading[]
-{ProductName} releases are available to download from {ReleaseDownload}. The release artefacts contain documentation, installation, and example `.yaml` files for deployment on {ProductPlatformName}. The installation, and example files are used throughout this documentation. Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
+To install {ProductName}, download the release artefacts from {ReleaseDownload}.
+
+The folder contains several YAML files to help you deploy the components of {ProductName} to {ProductPlatformName}, perform common operations, and configure your Kafka cluster. The YAML files are referenced throughout this documentation.
+
+Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
+
+The remainder of this chapter provides an overview of each component and short instructions for deploying the components to {ProductPlatformName} using the YAML files provided.
 endif::Downloading[]
 ifndef::Downloading[]
-The following procedures use installation and example YAML files to help you install {ProductName} on {ProductPlatformName}, perform common operations, and configure your Kafka cluster. {ExampleYAMLURL} are available to download from the AMQ Streams website.
+To install {ProductName}, download and extract the `install_and_examples.zip` file from the https://access.redhat.com/node/3596931/423/0[AMQ Streams website].
```

Comment:
The link will be different for GA. We should not forget to update it!

Reply:
I would not use a link to Red Hat web site in the upstream project as hard coded. Can we use an attribute for that link having it only downstream?

Reply:
+1 for using an attribute.

Reply:
> The link will be different for GA. We should not forget to update it!

Do we know what the link will be yet?

Reply:
Alright. I've added a new attribute named `{ZipDownload}`, which I'll define in the downstream-only repository. I can update the actual URL as soon as we know what it's going to be!